gulpfile.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var _ = require('lodash');
+var gulp = require('gulp');
+var argv = require('yargs').argv;
+var connect = require('gulp-connect');
+var header = require('gulp-header');
+var creative = require('./package.json');
+var uglify = require('gulp-uglify');
+var clean = require('gulp-clean');
+var webpackStream = require('webpack-stream');
+var webpack = require('webpack');
+var webpackConfig = require('./webpack.conf');
+
+var dateString = 'Updated : ' + (new Date()).toISOString().substring(0, 10);
+var banner = '/* <%= creative.name %> v<%= creative.version %>\n' + dateString + ' */\n';
+var port = 9999;
+
+gulp.task('serve', ['clean', 'build-dev', 'connect']);
+
+gulp.task('build', ['build-prod']);
+
+gulp.task('clean', () => {
+  return gulp.src(['dist/', 'build/'], {
+      read: false
+    })
+    .pipe(clean());
+});
+
+gulp.task('build-dev', () => {
+  return gulp.src(['src/creative.js'])
+    .pipe(webpackStream(webpackConfig))
+    .pipe(gulp.dest('build'));
+});
+
+gulp.task('build-prod', ['clean'], () => {
+  var cloned = _.cloneDeep(webpackConfig);
```

Comment:
why do you need to clone this?

Reply:
For prod we remove source-maps from webpack config. So we clone the object and do not modify original webpack config.